tests/backup/backup_restore_basic_test.go

```diff
@@ -3270,8 +3270,13 @@ var _ = Describe("{AlternateBackupBetweenNfsAndS3}", func() {
 				s3CloudCredUID = uuid.New()
 				s3BackupLocationUID = uuid.New()
 				backupLocationMap[s3BackupLocationUID] = s3BackupLocationName
-				err = CreateCloudCredential(provider, s3CloudCredName, s3CloudCredUID, orgID, ctx)
-				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", s3CloudCredName, orgID, "AWS"))
+				if provider == drivers.ProviderNfs {
```

Comment:
why we need to createcloudcredential if provider is nfs ?

Reply:
For Line 3280 ->CreateS3BackupLocation
In this testcase, we need to create both s3 and NFS backup location

Reply:
ok , so over implementation would be like we can test both alternate backup b/w s3 and nfs and vice-versa, correct ?

Reply:
yes